packages/runtime/src/slate/BlockPlugin/index.ts

```diff
@@ -1,15 +1,24 @@
 import { Editor, Transforms } from 'slate'
 import { setBlockKeyForDevice } from './setBlockKeyForDevice'
 import { clearBlockKeyForDevice } from './clearBlockKeyForDevice'
+import { wrapInline } from './wrapInline'
+import { unwrapInline } from './unwrapInline'
 import { ElementUtils } from '../utils/element'
 
 export const Block = {
-  setBlockKeyForDevice: setBlockKeyForDevice,
-  clearBlockKeyForDevice: clearBlockKeyForDevice,
+  setBlockKeyForDevice,
+  clearBlockKeyForDevice,
+  wrapInline,
+  unwrapInline,
 }
 
 export function withBlock(editor: Editor) {
   const { normalizeNode } = editor
+
+  editor.isInline = entry => {
+    return ElementUtils.isInline(entry)
+  }
```

Comment:
This is the main line that was missing. Not having it caused our inlines to be normalized as slate thought they were block level.